packages/react-query/src/__tests__/useQuery.test.tsx

```diff
@@ -2485,38 +2486,6 @@ describe('useQuery', () => {
     expect(queryCache.find({ queryKey: key })!.options.retryDelay).toBe(20)
   })
 
-  it('should batch re-renders', async () => {
```

Comment:
This test doesn't work anymore when we're force rendering

I'm guess we just have to trust the concurrency in React to do this in an optimal way?